src/throttler.interface.ts

```diff
@@ -0,0 +1,21 @@
+export interface ThrottlerOptions {
+  /**
+   * The amount of requests that are allowed within the ttl's time window.
+   */
+  limit?: number;
+
+  /**
+   * The amount of seconds of how many requests are allowed within this time.
+   */
+  ttl?: number;
+
+  /**
+   * The user agents that should be ignored. Checked against the `User-Agent` header
```

Comment:
@jmcdo29 Always start sentences with a capital and end with a tabstop aka dot.

The above should be changed to `The user agents that should be ignored (checked against the User-Agent header).`

Reply:
Must've happened when I was rebasing everything for the PR